hack/util-install-bundle.sh

```diff
@@ -18,6 +18,19 @@ metadata:
   name: build-pipeline-selector
 spec:
   selectors:
+    - name: Hermetic build (docker)
+      pipelineParams:
+        - name: hermetic
+          value: "true"
+        - name: prefetch-input
+          value: "gomod"
+      pipelineRef:
+        name: docker-build
+        bundle: ${BUNDLE}
+      when:
+        dockerfile: true
+        language: Go
+        projectType: test-hermetic-build
```

Comment:
aligned with the [test repo's devfile metadata](https://github.com/redhat-appstudio-qe/retrodep/blob/18f04edd59ba877f0d5a95eb894a772071c795bc/devfile.yaml#L3-L5)